crates/genesis/src/parmigiana.genesis.json

```diff
@@ -0,0 +1,144 @@
+{
+    "config": {
+      "chainId": 88888,
+      "homesteadBlock": "0x0",
+      "daoForkBlock": "0x0",
+      "daoForkSupport": false,
+      "eip150Block": "0x0",
+      "eip150Hash": null,
+      "eip155Block": "0x0",
+      "eip158Block": "0x0",
+      "byzantiumBlock": "0x0",
+      "constantinopleBlock": "0x0",
+      "petersburgBlock": "0x0",
+      "istanbulBlock": "0x0",
+      "muirGlacierBlock": "0x0",
+      "berlinBlock": "0x0",
+      "londonBlock": "0x0",
+      "arrowGlacierBlock": "0x0",
+      "grayGlacierBlock": "0x0",
+      "mergeNetsplitBlock": "0x0",
+      "shanghaiTime": "0x0",
+      "cancunTime": "0x0",
+      "pragueTime": "0x2623A",
```

Comment:
needs osaka added, along with its blob schedule

Reply:
prague should be 0 as well?